test/heat_conduction_1D.jl

```diff
@@ -27,18 +28,25 @@
     l = AsteroidThermoPhysicalModels.thermal_skin_depth(P, k, ρ, Cₚ)
     Γ = AsteroidThermoPhysicalModels.thermal_inertia(k, ρ, Cₚ)
 
-    thermo_params = AsteroidThermoPhysicalModels.thermoparams(
-        P       = P,
-        l       = l,
-        Γ       = Γ,
-        R_vis   = 0.0,
-        R_ir    = 0.0,
-        ε       = 1.0,
-        z_max   = 1.0,
-        n_depth = 101,
-    )
+    R_vis = 0.0  # Reflectance in visible light [-]
+    R_ir  = 0.0  # Reflectance in thermal infrared [-]
+    ε     = 1.0  # Emissivity [-]
+
+    z_max   = 1.0  # Depth of the lower boundary of a heat conduction equation [m]
+    n_depth = 101  # Number of depth steps
+    Δz = z_max / (n_depth - 1)  # Depth step width [m]
 
-    println(thermo_params)
+    thermo_params = AsteroidThermoPhysicalModels.ThermoParams(
+        P,
+        fill(l,     n_face),
+        fill(Γ,     n_face),
+        fill(R_vis, n_face),  
```

Comment:
These extra spaces can be removed.
```suggestion
        fill(R_vis, n_face),
```